pandas/core/groupby.py

```diff
@@ -534,19 +534,18 @@ def curried(x):
             except Exception:
                 try:
                     return self.apply(curried)
-                except Exception:
+                except Exception as e:
 
                     # related to : GH3688
                     # try item-by-item
-                    # this can be called recursively, so need to raise
-                    # ValueError
-                    # if we don't have this method to indicated to aggregate to
-                    # mark this column as an error
-                    try:
-                        return self._aggregate_item_by_item(name,
-                                                            *args, **kwargs)
-                    except (AttributeError):
-                        raise ValueError
+                    # this can be called recursively, so need to raise a
```

Comment:
I don't like this at all, what exactly are you trying to do here? why are you not simply defining ``.rank`` on the groupby methods and passing thru args?